include/TPP/Passes.td

```diff
@@ -388,7 +388,7 @@ def DefaultPipeline : Pass<"default-pipeline", "ModuleOp"> {
   ];
 }
 
-def SetSPIRVCapabilities : Pass<"set-spirv-capabilities", "ModuleOp"> {
+def SetSPIRVCapabilities : Pass<"tpp-set-spirv-capabilities", "ModuleOp"> {
```

Comment:
@adam-smnk is there a better name for this?

Reply:
I think it's fine as it's a common name for this kind of pass

@Devjiu could you also update the corresponding tests?

Reply:
Oh, these tests weren't executed because our CI only runs on CPU, so no failures.

Reply:
I found it in tpp-mlir/test/GPU/Vulkan/set-spirv-capabilities.mlir. Updated but not verified. Do I have to run the lit tests manually to run this test?

Reply:
GPU tests are disabled by default as that requires extra dependencies. And we don't have CI setup for Vulkan atm.
But it's fine, I'll rerun it locally.

Reply:
Thanks! @adam-smnk please resolve this conversation once the GPU tests pass.

Reply:
Thanks)

Reply:
Works fine, thanks 👍